### ISSUE TITLE
ActionMenu auto open on hover

### ISSUE THREAD

**Issue by PieterL75** [netbox-community/netbox#8666]
### NetBox version

v3.2-beta1

### Feature type

Change to existing functionality

### Proposed functionality

Make the 'actionmenu' open and close automaticcly on hover in/out

### Use case

A click less !
Better user experience

**Comment by jeremystretch** [comment]
I'm not sure I like this idea. Seems like it could rather distracting pretty quickly.

**Comment by PieterL75** [comment]
When creating the contextmenu javascript addon, I played with the 'click to open' and 'auto-open'..
In the end, I found it more friendly to use, with the box auto opening without having to click the icon.
At first, I did not had the icon next to the link, and just opened it when hovering over the link itself, but that was indeed distracting.
But with a small space icon/box and auto-open enabled, it became very intuitive.
